backend/src/main/kotlin/org/loculus/backend/utils/MetadataEntry.kt

```diff
@@ -125,8 +164,23 @@ fun revisionEntryStreamAsSequence(metadataInputStream: InputStream): Sequence<Re
                     )
                 }
 
+                var fastaIds: List<FastaId>? = null
+
+                if (addFastaIds) {
+                    val fastaId = record[fastaIdHeader]
+                    if (fastaId.isNullOrEmpty()) {
+                        throw UnprocessableEntityException(
+                            "A row in metadata file contains no $fastaIdHeader: $record",
+                        )
+                    }
+
+                    fastaIds = fastaId.split(Regex("[ ,]+"))
```

Comment:
We seem to prohibit ` ` and `,` in submissionId field if no fastaId column is present in header.

Intentional?